magda-web-client/src/Components/SearchFacets/FacetTemporal.js

```diff
@@ -29,10 +29,10 @@ class FacetTemporal extends Component {
     }
 
     componentWillMount() {
-        this.componentWillReceiveProps(this.props);
+        this.getDerivedStateFromProps(this.props);
     }
 
-    componentWillReceiveProps(nextProps) {
+    getDerivedStateFromProps(nextProps) {
```

Comment:
The idea is to return the state from `getDerivedStateFromProps` instead of using `setState` :)